### ISSUE TITLE
Installation fails due to conflicting urllib3 version

### ISSUE THREAD

**Issue by NeolithEra** [glucee/unblockyouku_gateway#3]
Hi, users are unable to run **_unblock_youku_gateway_** due to dependency conflict with _**urllib3**_ package. 
As shown in the following full dependency graph of **_unblock_youku_gateway_**, it directly requires _**urllib3>=1.24.2**_，while **requests==2.20.0** requires _**urllib3>=1.21.1,<1.25**_.

According to pip’s _“first found wins”_ installation strategy, _**urllib3==1.25.3**_ is the actually installed version. 
However, _**urllib3==1.25.3**_ does not satisfy _**urllib3>=1.21.1,<1.25**_.

### Dependency tree
```
unblock-youku-gateway-master
| +-certifi(version range:==2018.10.15)
| +-chardet(version range:==3.0.4)
| +-idna(version range:==2.7)
| +-requests(version range:==2.20.0)
| | +-certifi(version range:>=2017.4.17)
| | +-chardet(version range:<3.1.0,>=3.0.2)
| | +-idna(version range:>=2.5,<2.8)
| | +-urllib3(version range:>=1.21.1,<1.25)
| +-urllib3(version range:>=1.24.2)
```

Thanks for your help.
Best,
Neolith

**Comment by NeolithEra** [comment]
### Solution

1.	Fix your direct dependency to be urllib3>=1.21.1,<1.25. 

2.	Fix your direct dependency to be requests==2.22.0.

3.	Remove your direct dependency urllib3, and use urllib3 transitively introduced by requests.
Which solution do you prefer, 1, 2 or 3?

 I have checked these revisions will not affect your downstream projects now.
@glucee  Please let me know your choice. I can submit a PR to solve this issue.

**Comment by glucee** [comment]
Thx for your solutions, I think 1 or 2 should be fine. Please submit your PR and if everything works I will approve it. Thanks again.
